### PR TITLE
Remove delta usage from badger interactions

### PR DESCRIPTION
**Pull request by pattyshack**

### CODE REVIEW

File: engine/execution/state/bootstrap/bootstrap.go [hunk 1]
```diff
@@ -8,8 +8,8 @@ import (
 	"github.com/rs/zerolog"
 
 	"github.com/onflow/flow-go/engine/execution/state"
```
Comment (zhangchiqing) [inline engine/execution/state/bootstrap/bootstrap.go:10]: maybe fix these linter errors first?

**Review by SaveTheRbtz — APPROVED** [review]

**Review by janezpodhostnik — APPROVED** [review]

**Comment by pattyshack** [comment]
bors merge

**Comment by pattyshack** [comment]
bors merge

**Comment by pattyshack** [comment]
bors merge

**Pull request merged**